.github/workflows/ci.yml

```diff
@@ -26,11 +26,18 @@ jobs:
       NODE_OPTIONS: '--dns-result-order=ipv4first'
     steps:
     - uses: actions/checkout@v4
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
         cache: npm
 
-    - run: npm ci
-    - run: npm test
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Unit tests
+      run: npm test
+
+    - name: Integration test
+      run: ./markdown-link-check test/alive-links-only.md
```

Comment:
Maybe I should add examples using each the command flags.

Also noticed that the command reference in the readme does not match what we get from `--help`.

Reply:
I've been imagining this at the gym. Will add some more checks later.

Reply:
I think this addition will catch the worst, it really won't work clangers, but I think it will be worth reviewing again when unit test coverage is fuller.